### ISSUE TITLE
Question: Does this TalkBack fork connect to any Google servers?

### ISSUE THREAD

**Issue by realpixelcode** [GrapheneOS/talkback#30]

**Comment by thestinger** [comment]
It doesn't use Google servers. It's not something our patches change.

**Comment by realpixelcode** [comment]
Thanks!

**Comment by PatrykMis** [comment]
To be precise, this fork doesn't change upstream TalkBack behavior. Like upstream, this fork includes play-services-vision library for OCR which (by the way) doesn't work well for unlabeled icons, buttons etc. Play-services-vision may connect to Google servers (if understood well), but only while performing OCR actions.
